frontend/src/pages/projects/screens/spawner/storage/CreateNewStorageSection.tsx

```diff
@@ -3,13 +3,16 @@ import { Stack, StackItem } from '@patternfly/react-core';
 import { CreatingStorageObject, UpdateObjectAtPropAndValue } from '~/pages/projects/types';
 import PVSizeField from '~/pages/projects/components/PVSizeField';
 import NameDescriptionField from '~/concepts/k8s/NameDescriptionField';
+import { SupportedArea, useIsAreaAvailable } from '~/concepts/areas';
+import StorageClassSelect from './StorageClassSelect';
 
 type CreateNewStorageSectionProps = {
   data: CreatingStorageObject;
   setData: UpdateObjectAtPropAndValue<CreatingStorageObject>;
   currentSize?: string;
   autoFocusName?: boolean;
   menuAppendTo?: HTMLElement;
+  isEdit?: boolean;
```

Comment:
What does this flag represent exactly? Because it seems odd to have an edit flag on a component with `CreateNew` in the name.

Reply:
right. i will rename it to `disableStorageClassSelect`